numpy/core/src/umath/ufunc_object.c

```diff
@@ -4312,7 +4308,7 @@ PyUFunc_GenericReduction(PyUFuncObject *ufunc,
         }
         ret = (PyArrayObject *)PyUFunc_Reduceat(ufunc, mp, indices, out,
                                             axes[0], otype->type_num);
```

Comment:
Indentation.

Reply:
reformatted to put everything but ufunc on the next line with 8 spaces.